source/docs/software/commandbased/commands.rst

```diff
@@ -121,4 +121,27 @@ What about a more complicated case? Below is a drive command, from the same exam
       :linenos:
       :lineno-start: 5
 
-Notice that this command does not override ``isFinished()``, and thus will never end; this is the norm for commands that are intended to be used as default commands (and, as can be guessed, the library includes tools to make :ref:`this kind of command <docs/software/commandbased/convenience-features:RunCommand>` easier to write, too!).
+Notice that this command does not override ``isFinished()``, and thus will never end; this is the norm for commands that are intended to be used as default commands (and, as can be guessed, the library includes tools to make :ref:`this kind of command <docs/software/commandbased/builtins:RunCommand>` easier to write, too!).
+
+Command Properties
+------------------
+
+In addition to the four lifecycle methods described above, each ``Command`` also has two properties, the value of which is defined by the ``runsWhenDisabled`` and ``getInterruptBehavior`` methods respectively. In most cases, the default value is enough.
+
+runsWhenDisabled
+^^^^^^^^^^^^^^^^
+
+The ``runsWhenDisabled()`` method returns a ``boolean``/``bool`` specifying whether the command may run when the robot is disabled. With the default of returning ``false``, the command will be canceled when the robot is disabled and attempts to schedule it will do nothing. Returning ``true`` will allow the command to run and be scheduled when the robot is disabled.
+
+.. important:: Hardware outputs are disabled when the robot is disabled, regardless of ``runsWhenDisabled()``!
```

Comment:
The only exception being AddressableLED

Reply:
I think AddressableLED is enough of a niche that it doesn't warrant being mentioned. If the AddressableLED page mentions that it doesn't get disabled, I think that's sufficient. If you disagree, I'll add a parenthetical.